indigo_libs/indigo_client.c

```diff
@@ -472,6 +476,7 @@ indigo_result indigo_connect_server(const char *name, const char *host, int port
 	strncpy(indigo_available_servers[empty_slot].host, host, INDIGO_NAME_SIZE);
 	indigo_available_servers[empty_slot].port = port;
 	indigo_available_servers[empty_slot].socket = 0;
+	indigo_available_servers[empty_slot].interfaceIndex = interfaceIndex;
```

Comment:
I do not understand how this works, indigo_available_servers[empty_slot].interfaceIndex used anywhere? can you explain it?